.ci/packaging.groovy

```diff
@@ -198,17 +198,23 @@ def tagAndPush(name){
     tagName = "pr-${env.CHANGE_ID}"
   }
 
-  def oldName = "${DOCKER_REGISTRY}/beats/${name}:${libbetaVer}"
-  def newName = "${DOCKER_REGISTRY}/observability-ci/${name}:${tagName}"
-  def commitName = "${DOCKER_REGISTRY}/observability-ci/${name}:${env.GIT_BASE_COMMIT}"
   dockerLogin(secret: "${DOCKERELASTIC_SECRET}", registry: "${DOCKER_REGISTRY}")
-  retry(3){
-    sh(label:'Change tag and push', script: """
-      docker tag ${oldName} ${newName}
-      docker push ${newName}
-      docker tag ${oldName} ${commitName}
-      docker push ${commitName}
-    """)
+
+  // supported image flavours
+  def variants = ["", "-ubi8"]
```

Comment:
Add `-oss` too? Thoug I think they are not available for all beats.

Reply:
As this will add more logic to the tagAndPush method, let's do it in a follow-up, as this PR just fixes the ubi8 images, which is a requirement by the E2E tests.

Reply:
If that's the case then L211 should be changed otherwise it will fail.

Is there any place where the list of beats that support `ubi8`?